.github/workflows/pr-lint.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           PR_BODY=$(gh pr view ${{ github.event.pull_request.number }} --json body --jq ".body")
-          ISSUES=$(echo $PR_BODY | tr '[:upper:]' '[:lower:]' | sed -n -E '/(close|closes|closed|fix|fixes|fixed|resolve|resolves|resolved|relates to|related to|part of) #[0-9]+/p')
+          ISSUES=$(echo $PR_BODY | tr '[:upper:]' '[:lower:]' | sed -n -E '/((close|closes|closed|fix|fixes|fixed|resolve|resolves|resolved|relates to|related to|part of) (#|https:\/\/github.com\/opentffoundation\/opentf\/issues\/)[0-9]+)/p'
```

Comment:
sed allows swapping the `/` character with any other single character; this allows the `/` character to be used without escaping.

Consider updating to:
```
sed -n -E '\;((close|closes|closed|fix|fixes|fixed|resolve|resolves|resolved|relates to|related to|part of) (#|https://github.com/opentffoundation/opentf/issues/)[0-9]+);p'
```
The `\/` (slightly) complicates reading the URL.  Too, while it's unlikely to change, this allows us to maintain the URL without escaping its six backslashes.

I don't mean to be nit-picky, but since you're updating it anyways, I hope this is a useful addition.

And thanks for fixing this!  This exact issue bit me very recently.  : )

Reply:
updated
thank you